src/main.rs

```diff
@@ -68,29 +66,46 @@ fn main() {
 
 /// A Shop is a collection of 3 cards.
 struct Shop {
-    // TOOD: Add the field to this struct
+    cards: [Card; 3],
 }
 
 impl Shop {
     /// Get the price of the most expensive card in the shop
     fn most_expensive(&self) -> u32 {
-        todo!()
+        let mut h: u32 = 0;
```

Comment:
In the future, it's probably a good idea to use larger variable names :) (note, if you select it in VS Code and press F2, you'll be able to rename all uses of that value)